src/content/docs/usage/workspaces.mdx

```diff
@@ -88,11 +88,34 @@ __Prerequisites__
 __Procedure__
 1.  Run the following command:
     ```shell
-    $ daytona delete
+    daytona delete
     ```
-2.  Select the desired Workspace for deletion in the list and hit `Enter`.
+2.  Mark Workspaces for deletion by highlighting them and pressing `x`.
+    <Aside type="note">
+    Pressing `Enter` without using `x` to select individual Workspaces will select the currently highlighted Workspace for deletion.
+    </Aside>
+3.  Press `Enter`.
 3.  Confirm the action.
 
+### Non-Interactively
+You can delete 1 or more existing Workspaces non-interactively.
+
+__Prerequisites__
+*   A working installation of Daytona.
+*   At least 1 existing Workspace.
+
+__Procedure__
+1.  Run the following command where `WORKSPACE` is a space-separated list of 1 or more Workspace names:
+    ```shell
+    daytona delete <WORKSPACE, ...>
+    ```
+
+    __Example__
+    ```shell
+    daytona delete web-microservices erlang-test
```

Comment:
@osslate in the procedure there is a comma "," but in the example there is not. Which is correct?

Reply:
The comma is to signify that you can specify multiple Workspaces; I'll clarify it with a `__Syntax__` heading for that codeblock.

Reply:
As a reader i still find this confusing. is there or is there not a comma? If not remove it from the Procedure.

Suggestion merge Procedure and example into one - less confusing and straight to the point:

`daytona delete workspace_name_1 workspace_name_2`

or something like that

Reply:
Addressed in [fbdcaa2](https://github.com/daytonaio/docs/pull/61/commits/fbdcaa24701685759c92ae8748a3ac7d6f0bfca8)